elements/pfe-accordion/src/pfe-accordion.js

```diff
@@ -63,7 +78,9 @@ class PfeAccordion extends PFElement {
 
   static get events() {
     return {
-      change: `${this.tag}:change`
+      change: `${this.tag}:change`,
```

Comment:
Looking at the other components, it looks like we're using active state instead of passive.  I like the idea of passive for clarity but let's add an issue to do that as a breaking change across all components.  See pfe-jump-links-panel, pfe-modal, pfe-select, pfe-dropdown, pfe-cta.

Reply:
Agree.  Makes sense!